README.md

```diff
@@ -34,18 +34,18 @@ automatically use the latest OSM data. These are manually managed and hosted by
 Dustin on assets.od2net.org.
 
 If you're developing locally, you can avoid hitting od2net.org by setting up
-three directories in `web/public/`: `osm`, `boundaries`, and `cnt_demand`. If
-`web/public/osm/areas.json` exists, then the Svelte app will load from
-localhost, not od2net.org. You can copy from od2net.org to set this up,
+two directories in `web/public/`: `severance_pbfs` and `boundaries`. If
+`web/public/severance_pbfs/areas.json` exists, then the Svelte app will load
+from localhost, not od2net.org. You can copy from od2net.org to set this up,
 choosing what study areas to cache:
 
 ```
 AREAS="bristol edinburgh strasbourg ut_dallas"
 
 cd web/public
-mkdir boundaries osm cnt_demand
+mkdir boundaries severance_pbfs
 
-cd osm
```

Comment:
If you've got this cached locally, you'll need `mv osm severance_pbfs`. The naming is pretty awkward, but lots of other projects are currently pointing to it. I'll clean up my Cloudflare storage later and rename them all.

Reply:
(The new script needs to be updated too; if you merge first, I'm happy to rebase this and fix here